instructors.md

```diff
@@ -337,6 +228,132 @@ a substantial fraction won't be able to navigate to the right directory without
+    and everyone gives feedback on everyone (including themselves).
+    This feedback then goes into the Etherpad for discussion.
+
+*   It's important to record all three videos and then watch all three:
+    if the cycle is teach-review-teach-review,
+    the last person to teach runs out of time.
+    Doing all the reviewing after all the teaching
+    also helps put a bit of distance between the teaching and the reviewing,
+    which makes the exercise slightly less excruciating.
+
+*   This exercise only works if there are breakout rooms available:
+    if everyone is trying to record in the same room,
+    the audio cross-talk makes the recordings unintelligible.
+
+*   We use Etherpad for in-person training,
+    both for note-taking and for posting exercise solutions and feedback on recorded lessons
+    Questions and discussion are done aloud.
+
+### Two-Day Online With Groups
+
+*   We use Google Hangouts and Etherpad as in the multi-week version.
+    Each group of learners is together in a room using one camera and microphone,
+    rather than each being on the call separately.
+    We have found that having good audio matters more than having good video,
+    and that the better the audio,
+    the more learners can communicate with the instructor and other rooms by voice
+    rather than by using the Etherpad chat.
+
+*   We do the video lecture exercise as in the two-day in-person training.  
+
+<a name="schedule"></a>
+## Schedule
+
+The schedule for the course is evolving along with the content,
+but here's what we do in a typical two-day event:
+
+| Day 1 | 09:00 | Welcome: key concepts and training goals            |
+|       | 09:30 | Novice mind-set; formative vs. summative assessment |
+|       |       | *Exercise: multiple choice questions*               |
+|       | 10:30 | Coffee                                              |
+|       | 10:45 | Lesson study; giving feedback                       |
+|       |       | *Exercise: video recorded teaching*                 |
+|       | 11:45 | Morning wrap-up (minute cards)                      |
+|       | 12:00 | Lunch                                               |
+|       | 13:00 | Concept maps                                        |
+|       |       | *Exercise: draw and critique concept maps*          |
+|       | 14:15 | Coffee                                              |
+|       | 14:30 | Learning objectives                                 |
+|       |       | *Exercise: write and critique learning objectives*  |
+|       | 15:15 | Cognitive load                                      |
+|       | 15:30 | *Exercise: faded example*                           |
+|       | 15:45 | Wrap-up (one up, one down)                          |
+|       | 16:00 | Close                                               |
+|       |       | *Homework: read operations guide*                   |
+| Day 2 | 09:00 | Live coding demonstration                           |
```

Comment:
This needs **significantly more** time. Live coding demo is about 20-30 min in total: intro plus the demo itself which should be about 10-15 min so that the participants who have little experience with workshops get the gist. Then the exercise takes **a lot of time**. I give them up to 8-10 min each to teach a chunk of a lesson and record. This totals up to 45-50 min (groups setting up, swapping recording device etc.).